src/main/java/Spike.java

```diff
@@ -1,7 +1,14 @@
+import java.io.File;
```

Comment:
I like that you had import the functions explicitly, it makes it easier to see which classes are used.